app/utils/roles.js

```diff
@@ -20,5 +48,16 @@ module.exports = {
       }
     })
     return found
+  },
+  getAvailableRolesForService: agentInitiatedMotoEnabled => {
+    return Object.values(roles)
+      .filter(role => agentInitiatedMotoEnabled ? true : !role.agentInitiatedMotoServicesOnly)
```

Comment:
I think this logic could be simplified a bit, I think this is equivalent to `agentInitiatedMotoEnabled || !role.agentInitiatedMotoServicesOnly`